packages/ember-htmlbars/tests/attr_nodes/boolean_test.js

```diff
@@ -1,92 +0,0 @@
-import EmberView from 'ember-views/views/view';
-import run from 'ember-metal/run_loop';
-import { compile } from '../utils/helpers';
-import { equalInnerHTML } from 'htmlbars-test-helpers';
-
-var view;
-
-function appendView(view) {
-  run(function() { view.appendTo('#qunit-fixture'); });
-}
-
-QUnit.module('ember-htmlbars: boolean attribute', {
```

Comment:
This is now tested by [these tests](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/input-test.js)